flasc/data_processing/filtering.py

```diff
@@ -734,15 +790,21 @@ def get_power_curve(self, calculate_missing=True):
         return self.pw_curve_df
 
     def plot_farm_mean_power_curve(self, fm=None):
-        """Plot all turbines' power curves in a single figure. Also estimate
+        """Plot all turbines' power curves in a single figure.
```

Comment:
Is the main functionality here to plot all the curves, or to plot the mean curve? Perhaps it would be better to have the summary line say "Plot mean of all turbines' power curves and show individual curves." or similar

Reply:
That is better